packages/front-end/services/utils.tsx

```diff
@@ -42,3 +42,12 @@ export function getEvenSplit(n: number) {
 
   return weights;
 }
+
+export function isNullUndefinedOrEmpty(x) {
+  if (x === null) return true;
+  if (x === undefined) return true;
+  if (x === {}) return true;
+  if (x === []) return true;
```

Comment:
I don't think this works.  Objects and arrays are compared by reference, not value.

Reply:
```
let a = {};
let aa = { object: "" }
let b = [];
let bb = [1, 2, 3];

function isNullUndefinedOrEmpty(x) {
    if (x === null) return true;
    if (x === undefined) return true;
    if (!Object.keys(x).length) return true;
    if (x === "") return true;
    return false;
}

console.log(isNullUndefinedOrEmpty(a));
console.log(isNullUndefinedOrEmpty(aa));
console.log(isNullUndefinedOrEmpty(b));
console.log(isNullUndefinedOrEmpty(bb));
```

Let me know if you can think of edge cases to break this script. If not I fixed the issue